### PR TITLE
Nd loading icon cmp

### PR DESCRIPTION
**Pull request by npdarrington**
### Participating Group Members:

- Nathan Darrington

### Code Highlights:

- A LoadingIcon.vue component that can be used when the App has a loading period and we want better visual feedback for the user.

### Have Tests Been Added?

- [x] No.
- [ ] Yes, but all tests are not passing.
- [ ] Yes, and all are passing.

### Any background context you want to provide?

- The paw prints are animated and will scroll with keyframes. Still picture depicts what they look like before animation.

### Message/Questions for reviewer:

- N/A

### Issues:

- N/A

### Screenshots (if appropriate):

Screen Shot 2021-01-11 at 9 43 12 PM

### Tracking Consistency:

- [x] added appropriate labels
- [x] My code follows the code style of this project and has removed all unnecessary annotations
- [x] I have added comments on my pull request, particularly in hard-to-understand areas
- [ ] All new and existing tests passed

- [x] looked at PR preview to check spelling, syntax, formatting, and completion

### CODE REVIEW

**Review by stacyp2006 — APPROVED** [review]
Very cool. I love the pawprints. Thanks for taking this on.

File: src/components/LoadingIcon.vue [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+@keyframes lds-ellipsis1 {
+  0% {
+    transform: scale(0);
+  }
+  100% {
+    transform: scale(1);
+  }
+}
+
+@keyframes lds-ellipsis3 {
```
Comment (nicolegooden) [inline src/components/LoadingIcon.vue:65]: I haven't used `keyframes` before.  Looking forward to doing some research on my end to get a better understanding!

**Review by nicolegooden — APPROVED** [review]
Thanks for getting creative with the loading feature of our app! I'm excited to see what it looks like when it's implemented fully.

**Review by bretm9 — APPROVED** [review]
Thanks for Adding this! This is a nice feature that adds so much polish to the app!

**Pull request merged**